app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError(
+                "You need to be vaccinated to visit the cafe!"

```

Comment:
Code Style: Your error message is clear and descriptive, which is good. However, you can improve it by including the name of the visitor in the message. For example, `f'{visitor['name']}, you need to be vaccinated to visit the cafe!'`. This makes the error message more personal and informative.